components/form-token-field/index.js

```diff
@@ -314,6 +316,7 @@ class FormTokenField extends Component {
 
 	addNewToken( token ) {
 		this.addNewTokens( [ token ] );
+		this.speak( this.props.messages.added );
```

Comment:
Why abstract out `this.props.messages` instead of just including the message here?  `Item added` is generic enough to cover any situation.